assets/scss/partials/components/_notice.scss

```diff
@@ -0,0 +1,51 @@
+.notice {
+  padding-left: 0;
+  padding-right: 0;
+  width: 100%;
+  @include themed() {
+    background-color: t('notice-info');
+  }
+
+  &--update {
+    @include themed() {
+      background-color: t('notice-update');
+    }
+  }
+  &--warning {
+    @include themed() {
+      background-color: t('notice-warning');
+    }
+  }
+
+  &__title {
+    background-color: rgb(96, 134, 180);
+    float: right;
+    font-weight: 300;
+    letter-spacing: 0.025em;
+    margin-bottom: 0.25rem;
+    margin-left: 1rem;
+    padding-left: 1rem;
+    padding-right: 1rem;
+    @include themed() {
+      color: t('accent');
+    }
+
+    &--update {
+      background-color: rgb(81, 216, 138);
+    }
+    &--warning {
+      background-color: rgb(239, 87, 83);
+    }
+  }
+
+  &__content {
+    margin-bottom: 1.5rem;
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 100%;
+    padding-left: 1rem;
+    padding-right: 1rem;
+    padding-top: 8px;
+    padding-bottom: 8px;
```

Comment:
Please use shorthands in general. I'm already counting the lines you could save. :smile: